### DISCUSSION TITLE
Is there a group size limit for groups ?

### DISCUSSION THREAD

**Discussion by 0x8mx** [aahnik/tgcf#199] [Q&A] [answered]
Hey, 
is there a group size limit for groups ?
For example forwarding from a big group with 3000+ members?

Iam very excited for the regex function. Furthermore the bot is awesome , keep up! 💪

**Comment by aahnik** [comment, marked as answer]
No there is no limit. The group size does not matter. Your user account/bot which you are using to forward messages must be a member of the group.
